src/components/settings/nontandem/NonTandem.js

```diff
@@ -0,0 +1,250 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+
+import Header from '../common/Header';
+import Table from '../common/Table';
+import CollapsibleContainer from '../common/CollapsibleContainer';
+
+import * as constants from '../../../utils/constants';
```

Comment:
I changed it. I think I may have c&ped from another file where it was done the other way, so at some point we may want to do a pass for consistency...